### ISSUE TITLE
Unable to move shaker with Jaws

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/beers-law-lab#381]
**Device**
Samsung
**OS**
Win 11
**Browser**
Chrome + Jaws
**Problem Description**
For https://github.com/phetsims/qa/issues/1281
I was unable to move the shaker on the Concentration screen using Jaws and keyboard nav. When focused on it, it would say "movable" and when trying to move it with the arrow keys it would say "dispensing" but nothing happened.

**Comment by pixelzoom** [comment]
Yes, closing this issue.  Thanks all.
